### ISSUE TITLE
Inconsistent conversion of datetime slang

### ISSUE THREAD

**Issue by moin18** [kennethreitz/maya#36]
I am in `Asia/Kolkata` timezone and one of the test is failing due to time-zone conversion.

> ===================================================================================== FAILURES ======================================================================================
> __________________________________________________________________________________ test_slang_date __________________________________________________________________________________
> 
>     def test_slang_date():
>         d = maya.when('tomorrow')
> >       assert d.slang_date() == 'tomorrow'
> E       assert 'today' == 'tomorrow'
> E         - today
> E         + tomorrow
> 
> test_maya.py:77: AssertionError

Related information when I ran `dateparse.parse()` and `maya.when()` in python shell:
```

>>> t = dateparser.parse('tomorrow', settings={'TIMEZONE': 'UTC', 'RETURN_AS_TIMEZONE_AWARE': True, 'TO_TIMEZONE': 'UTC'})
>>> d = maya.when('tomorrow')
>>> t
datetime.datetime(2016, 12, 27, 20, 5, 33, 716383, tzinfo=<StaticTzInfo 'UTC'>)
>>> d.datetime()
datetime.datetime(2016, 12, 27, 20, 5, 35, 304214, tzinfo=<UTC>)
>>> d.slang_date()
'today'
>>> d.local_timezone
'Asia/Kolkata'
```

**Comment by moin18** [comment]
Using local timezone to get the `slang_date`. Similar behavior is followed in `slang_time` function: https://github.com/kennethreitz/maya/blob/master/maya.py#L220

PR: https://github.com/kennethreitz/maya/pull/38

**Comment by kennethreitz** [comment]
Fixed in #38
